bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/context.js

```diff
@@ -1,7 +1,7 @@
 import { pt, pb, pi, pn } from '../helpers.js'
 
 export default () => [
-  pt('functions', 'Widget Functions', 'Object with key:arrow-function pairs. Functions are available to expressions in all child components via the <code>fn</code> object.'),
-  pt('constants', 'Widget Constants', 'Object with key:constant pairs. Constants are available to expressions in all child components via the <code>const</code> object.'),
-  pt('variables', 'Widget Variables', 'Object with key:variable default value pairs. Variables are available to expressions in all child components via the <code>vars</code> object and take precedence over variables with the same name from higher contexts.')
+  pt('functions', 'Widget Functions', 'Object with key:arrow-function pairs. Functions are available to expressions in all child components via the <code>fn</code> object.<br />Functions are evaluated on every call and re-evaluated when dependent values change.'),
+  pt('constants', 'Widget Constants', 'Object with key:constant pairs. Constants are available to expressions in all child components via the <code>const</code> object.<br />Constants are evaluated only on widget build and cannot be changed.'),
```

Comment:
```suggestion
  pt('constants', 'Widget Constants', 'Object with key:constant pairs. Constants are available to expressions in all child components via the <code>const</code> object.<br />Constants are evaluated once before the widget is displayed the first time and cannot be changed.'),
```

Whilst my suggestion is technically still not 100% correct I think it is a good compromise between technical correctness and being easy to understand for users.